lib/Standalone/TransformUtils.cpp

```diff
@@ -119,15 +118,14 @@ static Value getSliceOperand(OpBuilder &builder, linalg::LinalgOp linalgOp,
                                               reducedType.cast<MemRefType>(),
                                               operand, offsets, sizes, strides);
 
-  assert(extractOperation->getNumResults() == 1);
+  assert(extractOperation->getNumResults() == 1 && "expect single result");
   return extractOperation->getResult(0);
 }
 
-static Value
-getSliceOperandImpl(OpBuilder &builder, linalg::LinalgOp linalgOp,
-                    OpOperand *operand, ValueRange ivs, ValueRange valuesToUse,
-                    unsigned desiredResultRank,
-                    const SmallVector<OpFoldResult> &sizesToUse = {}) {
+static Value getSliceOperandImpl(OpBuilder &builder, linalg::LinalgOp linalgOp,
```

Comment:
This doesn't seem to be called anymore...

Reply:
It is called by `getSliceOperand`

Reply:
Ignore me, Github folding code